TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -453,7 +459,64 @@ internal void Compare(IEnumerable<Program> programs, IEnumerable<TypeCobol.Compi
             ParserUtils.CheckWithResultReader(paths.SamplePath, result, expected, expectedResultPath);
         }
     }
+    /// <summary>
+    /// Selects the SQL code elements
+    /// </summary>
+    internal class SqlComparator : FilesComparator
+    {
+        private class ASTVisitor : AbstractAstVisitor 
+        {
+            private readonly StringWriter _writer;
+            private int line = 1;
+            public ASTVisitor(StringBuilder builder)
+            {
+                _writer = new StringWriter(builder);
+            }
+
+            public override bool Visit(SelectStatement selectStatement)
+            {
+                _writer.WriteLine("line" + line + ":SelectStatement");
+                line++;
```

Comment:
line is not an increment, this is a real data read from CodeElement, use the `Line` property and remove the `line` field.